### PR TITLE
User story3

### PR DESCRIPTION
**Pull request by anotherMichaelMaier**
I got lost in the process and added a "details button" already that is included in US 4, so I'm sorry for that and would appreciate if you could focus on the navbar feature.

### CODE REVIEW

File: components/AddCard.js [hunk 1]
```diff
@@ -62,10 +62,14 @@ export default function AddCard({ createTask }) {
...
 `;
 
 const StyledForm = styled.form`
   display: flex;
   flex-direction: column;
 `;
+
+const StyledButton = styled.button`
+  display: flex;
+  text-align: center;
```
Comment (RobinWitt) [inline components/AddCard.js:74]: Hi Michael, 
display: flex isn't really necessary when you are using text-align.
Option A: `text-align: center`
Option B: `display: flex, justify-content: center, align-items: center`
Depending on what is going to be inside your button.

File: components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import Link from "next/link";
+import TodoIcon from "@/public/todoIcon.svg";
+import ProgressionIcon from "@/public/progressIcon.svg";
+import DoneIcon from "@/public/doneIcon.svg";
+export default function Navbar() {
+  return (
+    <StyledNavbar>
```
Comment (RobinWitt) [inline components/Navbar.js:8]: Would be nice to have some kind of indicator to show which Page I am currently on. I assume you are going to implement that later :)

**Review by RobinWitt — APPROVED** [review]
Navbar is working, but I would really like to see some indication as I said in the comment.
Also, clicking on your 'details' button opens up the details of every item. But since the navbar is working, you have my approval!

File: components/AddCard.js (hunk 1 above)
Comment (anotherMichaelMaier) [reply components/AddCard.js:74]: Thanks, will change that asap! Appreciate it.

File: components/Navbar.js (hunk 2 above)
Comment (anotherMichaelMaier) [reply components/Navbar.js:8]: Yes I just added some basic styling for now.

**Pull request merged**